### PR TITLE
Refactor/entity folders

### PR DESCRIPTION
**Pull request by Raycoms**
Closes #
Closes #
Closes #

# Changes proposed in this pull request:
- Rearrange entity folders
-
-

image

[ ] Yes I tested this before submitting it.
[ ] I also did a multiplayer test.

Review please

### CODE REVIEW

**Comment by MotionlessTrain** [comment]
There seems to be a typo in one of the folders: `.../worker/production/agrilculture`

File: src/main/java/com/minecolonies/core/entity/ai/workers/crafting/EntityAIWorkAlchemist.java [hunk 1]
```diff
@@ -24,7 +24,7 @@
 import com.minecolonies.core.colony.buildings.workerbuildings.BuildingAlchemist;
 import com.minecolonies.core.colony.interactionhandling.StandardInteraction;
 import com.minecolonies.core.colony.jobs.JobAlchemist;
-import com.minecolonies.core.entity.ai.basic.AbstractEntityAICrafting;
+import com.minecolonies.core.entity.ai.workers.crafting.AbstractEntityAICrafting;
```
Comment (MotionlessTrain) [inline src/main/java/com/minecolonies/core/entity/ai/workers/crafting/EntityAIWorkAlchemist.java:27]: This import is technically not needed anymore (as the class is now in the same package). Same for most of the other AI classes in this package

File: src/main/java/com/minecolonies/core/entity/citizen/EntityCitizen.java [hunk 2]
```diff
@@ -48,14 +48,16 @@
...
+import com.minecolonies.core.entity.citizen.citizenhandlers.*;
+import com.minecolonies.core.entity.other.SittingEntity;
+import com.minecolonies.core.entity.ai.workers.AbstractEntityAIBasic;
+import com.minecolonies.core.entity.ai.workers.CitizenAI;
+import com.minecolonies.core.entity.ai.workers.guard.AbstractEntityAIGuard;
 import com.minecolonies.core.entity.ai.minimal.EntityAICitizenChild;
 import com.minecolonies.core.entity.ai.minimal.EntityAIInteractToggleAble;
 import com.minecolonies.core.entity.ai.minimal.LookAtEntityGoal;
-import com.minecolonies.core.entity.citizen.citizenhandlers.*;
+import com.minecolonies.core.entity.entity.citizen.citizenhandlers.*;
```
Comment (MotionlessTrain) [inline src/main/java/com/minecolonies/core/entity/citizen/EntityCitizen.java]: These imports look strange to me. Was there already an `entity` package within `entity`?

**Review by MotionlessTrain — APPROVED** [review]
Just to be sure: These refactors don't have issues with deserialising NBT?
(as the previous refactor PR had code to make sure that the classes could still be found when their name was read from NBT)

**Review by marchermans — DISMISSED** [review]
Not really sure what to review here. But I approve!

**Comment by Raycoms** [comment]
> Not really sure what to review here. But I approve!

Yeah it's more if you agree with the structure xD

**Review by someaddons — DISMISSED** [review]

**Pull request merged**